mleap-core/src/test/scala/ml/combust/mleap/core/types/CastingSpec.scala

```diff
@@ -132,10 +141,10 @@ class CastingSpec extends FunSpec {
   for(((from, to, fromValue, expectedValue), i) <- castTests.zipWithIndex) {
     describe(s"cast from $from to $to - $i") {
       it("casts the scalar") {
-        val c = Casting.cast(ScalarType(from), ScalarType(to)).get.get
-        val oc = Casting.cast(ScalarType(from), ScalarType(to).nonNullable).get.get
-        val co = Casting.cast(ScalarType(from).nonNullable, ScalarType(to)).get.get
-        val oco = Casting.cast(ScalarType(from), ScalarType(to)).get.get
+        val c = Casting.cast(ScalarType(from), ScalarType(to)).getOrElse(Success((v: Any) => v)).get
```

Comment:
This just says that when `Casting.cast` returns `None` (indicating that no cast is necessary), then the tests should use the identity function instead